S4TF_Tutorial_3.ipynb

```diff
@@ -1764,7 +1763,7 @@
         "Experiment with different models and see how the accuracy results differ. In particular change the following parameters:\n",
         "*   Set training epochs set to 1\n",
         "*   Number of neurons in the Dense layer following the Flatten one. For example, go really low (e.g. 10) in ranges up to 512 and see how accuracy changes\n",
-        "*   Add additional Dense layers between the Flatten and the final Dense(10,  activation = Softmax), experiment with different units in these layers\n",
+        "*   Add additional Dense layers between the Flatten and the final Dense(10, activation = Softmax), experiment with different units in these layers\n",
         "*   Don't normalize the pixel values, and see the effect that has\n",
         "\n",
         "\n",
```

Comment:
Below this: "Remember to enable GPU" should maybe be "Remember to enable GPU acceleration"